tests/integration/targets/lookup_passwordstore/tasks/package.yml

```diff
@@ -31,6 +31,29 @@
       disable_gpg_check: yes
   when: ansible_facts.pkg_mgr in ['zypper', 'community.general.zypper']
 
+# See https://github.com/gopasspw/gopass/issues/1849#issuecomment-802789285
+- name: Install on Debian
+  when: ansible_facts.os_family == 'Debian'
+  become: yes
+  block:
+  - name: Fetch gopass repo keyring
+    shell: curl -o /usr/share/keyrings/gopass-archive-keyring.gpg \
+      https://packages.gopass.pw/repos/gopass/gopass-archive-keyring.gpg
+  - name: Add gopass repo
+    command: tee /etc/apt/sources.list.d/gopass.sources
+    args:
+      stdin: |
+        Types: deb
+        URIs: https://packages.gopass.pw/repos/gopass
+        Suites: stable
+        Architectures: amd64 arm64 armhf
+        Components: main
+        Signed-By: /usr/share/keyrings/gopass-archive-keyring.gpg
+  - name: Update repo
+    command: apt update
+  - name: Install package from gopass repo
+    command: apt install -y gopass
```

Comment:
Why not use the [apt](https://docs.ansible.com/ansible/devel/collections/ansible/builtin/apt_module.html#ansible-collections-ansible-builtin-apt-module) and [apt_repository](https://docs.ansible.com/ansible/devel/collections/ansible/builtin/apt_repository_module.html#ansible-collections-ansible-builtin-apt-repository-module) modules for this?